app/views/candidate_interface/equality_and_diversity/edit_sex.html.erb

```diff
@@ -6,10 +6,18 @@
     <%= form_with model: @sex, url: candidate_interface_edit_equality_and_diversity_sex_path, method: :patch do |f| %>
       <%= f.govuk_error_summary %>
 
-      <%= f.govuk_radio_buttons_fieldset :sex, caption: { text: t('equality_and_diversity.title'), size: 'xl' }, legend: { text: t('equality_and_diversity.sex.title'), size: 'xl', tag: 'h1' } do %>
+      <span class='govuk-caption-l'><%= t('equality_and_diversity.title')%></span>
+      
+      <h1 class='govuk-heading-l'><%= t('equality_and_diversity.sex.heading')%></h1>
```

Comment:
Should we swap these variable names around maybe, so that this one is the `title` (and also used in the `<title>` tag) and the other one is the `heading` (or possibly `question`?).

Reply:
I've changed heading to title but retained the other title as that locale is used across the flow, i can change all of them to heading/question if you'd prefer but it may make less sense on other views e.g disability